httpx/middleware/request_id.go

```diff
@@ -42,6 +42,13 @@ func (h *RequestID) ServeHTTPContext(ctx context.Context, w http.ResponseWriter,
 	return h.handler.ServeHTTPContext(ctx, w, r)
 }
 
+func AddRequestIDToCtx(h httpx.Handler) httpx.Handler {
```

Comment:
This duplicates the behavior of [ExtractRequestId](https://github.com/remind101/pkg/blob/chaining_middleware/httpx/middleware/request_id.go#L26), no? `ServeHTTPContext` defaults the the DefaultRequestIdExtract if none is provided.